src/DeviceInterface.js

```diff
@@ -38,7 +41,31 @@ const createAttachTooltip = (getAutofillData, refreshAlias, addresses) => (form,
 let attempts = 0
 
 class InterfacePrototype {
+    // Data
```

Comment:
We're now storing data right in the interface to simplify (and make less async) interactions with consumers.